### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by maartenvanderhoef**
Initial commit of a boilerplate micro service module which allows easy creating of typical AWS resources typical Microservices need.

### CODE REVIEW

File: iam.tf [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  statement {
+    sid    = "ListDynamoDB"
+    effect = "Allow"
+
+    resources = [
+      "*",
+    ]
+
+    actions = [
+      "dynamodb:List*",
```
Comment (cytopia) [inline iam.tf]: This permission requires explanation. Why does access to one DynamoDB instance imply those kinds of permissions to all other available DynamoDB instances

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+variable "rds_node_type" {
+  description = "VM type which should be taken for nodes in the RDS instance"
+  type        = "string"
+  default     = "db.t3.micro"
+}
+
+variable "rds_multi_az" {
+  description = "Replication settings"
+  type        = "string"
+  default     = true
```
Comment (cytopia) [inline variables.tf]: There is mixed usage of `true` and `"true"`. Let's please consolidate to use one or the other

Comment (danvaida) [reply variables.tf]: @cytopia I believe that in some of these cases (if there's an urgent need for something to be delivered), we can make these changes ourselves given that we're on common ground. WDYT?

File: README.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ module "ms_sample_redis" {
 
 ```hcl
 module "ms_sample_rds" {
-  source = "../../"
+  source = "github.com/flaconi/terraform-aws-microservice"
```
Comment (danvaida) [inline README.md]: This should be pinned to a specific version.

Comment (danvaida) [reply README.md]: Oh, it's the README, nvm.

File: iam.tf (hunk 1 above)
Comment (maartenvanderhoef) [reply iam.tf]: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_dynamodb_specific-table.html Can we leave it as is for now ?

File: variables.tf (hunk 2 above)
Comment (maartenvanderhoef) [reply variables.tf]: Valid point. Difference between Anton's modules and Cloudposse is that Cloudposse at the moment is trying to have fixed typing to be ahead with the 0.12 migration game.

**Comment by cytopia** [comment]
@maartenvanderhoef let me know when this is ready to be reviewed again.

**Comment by maartenvanderhoef** [comment]
> @maartenvanderhoef let me know when this is ready to be reviewed again.

@cytopia Ready for review.

**Pull request merged**